lib/reply.js

```diff
@@ -80,8 +80,22 @@ Reply.prototype.send = function (payload) {
   onSendHook(this, payload)
 }
 
+Reply.prototype.getHeader = function (key) {
+  return getHeader(this, key)
+}
+
+Reply.prototype.hasHeader = function (key) {
+  return Object.keys(this._headers).indexOf(key.toLowerCase()) > -1
```

Comment:
You can use `this._headers[key.toLowerCase()] !== undefined`, it will be faster.

Reply:
This was when I was still concocting ideas around supporting generic multiple headers. This should definitely be changed.